programs/ssl/ssl_client2.c

```diff
@@ -2083,13 +2083,6 @@ int main( int argc, char *argv[] )
     else
         mbedtls_printf( "    [ Record expansion is unknown ]\n" );
 
-#if defined(MBEDTLS_SSL_MAX_FRAGMENT_LENGTH)
-    mbedtls_printf( "    [ Maximum input fragment length is %u ]\n",
-                    (unsigned int) mbedtls_ssl_get_input_max_frag_len( &ssl ) );
-    mbedtls_printf( "    [ Maximum output fragment length is %u ]\n",
-                    (unsigned int) mbedtls_ssl_get_output_max_frag_len( &ssl ) );
-#endif
-
```

Comment:
Tests are looking for this output, [causing failures](https://jenkins-mbedtls.oss.arm.com/blue/organizations/jenkins/mbed-tls-pr-merge/detail/PR-4382-merge/10/pipeline/314) like:
```
Max fragment length: enabled, default .................................. FAIL
  ! pattern 'Maximum input fragment length is 16384' MUST be present in the Client output
  ! outputs saved to o-XXX-157.log
  ```
  I think it may be better to restore these lines and import `ssl_misc.h` from `ssl_test_lib.h`

Reply:
Same for `ssl_server2.c` as well.

Reply:
Thanks for spotting this @bensze01. I suggest to make this some internal debugging output.

Reply:
On second thought, I re-introduced the code in `ssl_client2.c/ssl_server2.c`, but now using the new record payload API.